### PR TITLE
Exclude instrumenting other web frameworks unless explictly specified

### PR DESCRIPTION
**Pull request by vaidy4github**

### CODE REVIEW

File: librato_python_web/instrumentor/__init__.py [hunk 1]
```diff
@@ -83,8 +84,20 @@
...
+        libs = general.get_option('libraries')
+        logger.info("Specified libraries = %s", libs)
+
+        integration = general.get_option('integration', 'django')
+        logger.info("Integration = %s", integration)
+
+        if not libs:
+            # By default, let us exclude the other web frameworks
+            # The user can pull in multiple frameworks by being explicit
+            libs = [lib for lib in _instrumentors.keys() if lib not in _web_fxes or lib == integration]
```
Comment (boxcongit) [inline librato_python_web/instrumentor/__init__.py]: So django and non-web frameworks are enabled by default, right?

Comment (swilibc) [inline librato_python_web/instrumentor/__init__.py]: So django and non-web frameworks are enabled by default, right?

Comment (vaidy4github) [inline librato_python_web/instrumentor/__init__.py]: Correct.

**Comment by bowillyard** [comment]
slack is down, so I am going to merge this one

**Pull request merged**